slides/boosting/rsrc/boosting_animation_multiclass.R

```diff
@@ -0,0 +1,14 @@
+library(mlr3verse)
```

Comment:
What's this file doing here? :)

Reply:
Sorry, I accidentally pushed a file I have just created also.
I reverted that action.
Please check again. Thank you :D